lib/project/match-view.coffee

```diff
@@ -10,16 +10,33 @@ class MatchView extends View
     range = Range.fromObject(match.range)
     matchStart = range.start.column - match.lineTextOffset
     matchEnd = range.end.column - match.lineTextOffset
-    prefix = removeLeadingWhitespace(match.lineText[0...matchStart])
+    prefix = match.lineText[0...matchStart]
     suffix = match.lineText[matchEnd..]
+    contextBefore = match.contextBefore or []
+    contextAfter = match.contextAfter or []
 
-    @li class: 'search-result list-item', =>
-      @span range.start.row + 1, class: 'line-number text-subtle'
-      @span class: 'preview', outlet: 'preview', =>
-        @span prefix
-        @span match.matchText, class: 'match highlight-info', outlet: 'matchText'
-        @span match.matchText, class: 'replacement highlight-success', outlet: 'replacementText'
-        @span suffix
+    liStyle = if match.CONTEXT_LINES > 0 then 'padding-top: 0; padding-bottom: 0'
+    divStyle = 'white-space: pre'
```

Comment:
Can this and the above style be placed in the `styles/` directory?  Maybe not `liStyle` because of the conditional but `divStyle` looks doable.